### PR TITLE
Introduced applicationName

### PR DESCRIPTION
**Pull request by wolfy2k**
Introduced applicationName for android, ios and windows.
Hopefully also fixed windows bundleId.
Tested only on android and ios.

### CODE REVIEW

**Comment by appden** [comment]
It would be really nice to see this merged. Thanks!

**Comment by GantMan** [comment]
Please update changelog also

File: android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java [hunk 1]
```diff
@@ -125,17 +126,21 @@ public String getCarrier() {
...
     constants.put("buildNumber", 0);
 
     try {
+      PackageInfo packageInfo = packageManager.getPackageInfo(packageName, 0);
       PackageInfo info = packageManager.getPackageInfo(packageName, 0);
       constants.put("appVersion", info.versionName);
       constants.put("buildNumber", info.versionCode);
       constants.put("firstInstallTime", info.firstInstallTime);
       constants.put("lastUpdateTime", info.lastUpdateTime);
+      constants.put("appName", packageManager.getApplicationLabel(applicationInfo).toString());
```
Comment (alvarezloaiciga) [inline android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java]: I am getting undefined applicationInfo, this is probably just `info`

Comment (wolfy2k) [reply android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java]: Hmm, probably happened in a commit which was resolving the conflicts. I was not careful enough. Can anyone make quick fix or do I need to create another pull request?

Comment (alvarezloaiciga) [reply android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java]: yeah https://github.com/rebeccahughes/react-native-device-info/pull/295

**Pull request merged**